packages/flutter/test/services/text_input_test.dart

```diff
@@ -15,15 +15,33 @@ void main() {
       expect(configuration.actionLabel, null);
     });
 
-    test('serializes to JSON', () async {
+    test('text serializes to JSON', () async {
       const TextInputConfiguration configuration = const TextInputConfiguration(
-        inputType: TextInputType.number,
-        obscureText: true,
-        autocorrect: false,
-        actionLabel: 'xyzzy'
+          inputType: TextInputType.text,
+          obscureText: true,
+          autocorrect: false,
+          actionLabel: 'xyzzy'
```

Comment:
fwiw: if you're using dartfmt, it usually does a better job when you give trailing commas